Expecto/Expecto.fs

```diff
@@ -647,9 +647,34 @@ module Impl =
                 "( ರ Ĺ̯ ರೃ )"
               else
                 ""
+          let commonAncestor (parentNames : string list) =
+            let rec loop ancestor (descendants : string list) =
+              match descendants with
+              | [] -> ancestor
+              | hd::tl when hd.StartsWith(ancestor)->
+                loop ancestor tl
+              | _ ->
+                if ancestor.Contains("/") then
+                  loop (ancestor.Substring(0, ancestor.LastIndexOf "/")) descendants
+                else
+                  "miscellaneous"
+
+            match parentNames with
+            | [x] -> x
+            | hd::tl ->
+              loop hd tl
+            | _ -> "miscellaneous" //we can't get here
+
           logger.logWithAck Info (
-            eventX "EXPECTO! {total} tests run in {duration} – {passes} passed, {ignores} ignored, {failures} failed, {errors} errored. {spirit}"
+            eventX "EXPECTO! {total} tests run in {duration} for {name} – {passes} passed, {ignores} ignored, {failures} failed, {errors} errored. {spirit}"
             >> setField "total" (List.sumBy (fun (_,r) -> r.count) summary.results |> commaString)
+            >> setField "name" (summary.results
```

Comment:
I don't like logging that does parsing conditional on the logging level (at least if you do an eta-expansion making `level` explicit; do you think you could extract the name to a field above?

Reply:
OK, I think I know what you are asking for (but then in a comment below you say "looking good"??)
I don't know if I will be able to change it today. Hopefully in a couple days.